spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -2097,9 +2097,17 @@ The preceding example overrides the default factory, and it should be applied to
 
 [[howto-batch-applications]]
 == Batch Applications
-This section answers questions that arise from using Spring Batch with Spring Boot.
+A number of questions often arise when people use Spring Batch from within a Spring Boot application. This section addresses the following concerns:
 
-NOTE: By default, batch applications require a `DataSource` to store job details.
+* <<howto-spring-batch-specifying-a-data-source>>
```

Comment:
We usually don't list sections like that. Our toc on the left hand side offers a list of sub-sections anyway so I think this is redundant (and likely to be outdated if we forget to update the list).

Reply:
It's regarded as a best practice (part of information mapping) in the technical writing community. However, I removed it, because I have no idea how it works with the ToC being constantly present. Do readers refer to the ToC as they read? (I don't, but I'm not everyone.) I wish I had data around that, to have something other than intuition on which to base this kind of decision. Anyway, I removed it.